crates/adapters/src/controller/stats.rs

```diff
@@ -715,6 +738,9 @@ pub struct OutputEndpointMetrics {
     pub transmitted_records: AtomicU64,
     pub transmitted_bytes: AtomicU64,
 
+    pub queued_records: AtomicU64,
```

Comment:
should there be some documentation explaining the difference between queued and buffered?

Reply:
will add